file.go

```diff
@@ -61,3 +68,12 @@ func FileFromJSON(bs []byte) (*File, error) {
 	}
 	return file, nil
 }
+
+type FilePropertyFunc func(*File)
+
+// FileDirection specify that the file is for incoming or outgoing
+func FileDirection(isIncoming bool) FilePropertyFunc {
```

Comment:
Is it more clear to have `OutgoingFile() FilePropertyFunc` and `IncomingFile()` ?